server/lobbyconnection.py

```diff
@@ -996,6 +1008,11 @@ async def command_game_join(self, message):
 
         await self.launch_game(game, is_host=False)
 
+    async def send_game_join_failed_response(self, new_message, legacy_message):
+        await self.send(new_message)
+        # For backwards compatibility
+        await self.send(legacy_message)
```

Comment:
I think I would not make a new method for this. Just call the `self.send` twice in each place. We should also add a comment that says `# DEPRECATED: use 'game_join_failed' instead` above each of the notice messages. That will help me to remember to remove those on the v2 branch because I can just search for `DEPRECATED` in the whole project.